src/trio/_util.py

```diff
@@ -395,9 +402,11 @@ def name_asyncgen(agen: AsyncGeneratorType[object, NoReturn]) -> str:
 
 # work around a pyright error
 if TYPE_CHECKING:
-    Fn = TypeVar("Fn", bound=Callable[..., object])
+    # Explicit "Any" is not allowed
+    Fn = TypeVar("Fn", bound=Callable[..., object])  # type: ignore[misc]
 
-    def wraps(
+    # Explicit "Any" is not allowed
```

Comment:
```suggestion
    # Explicit .../"Any" is not allowed
```

Reply:
I think handled in 50adccbe4338ade6638d286aea31f5f7aed2faf2